docker-compose.override.yml

```diff
@@ -4,13 +4,23 @@
 version: '3.6'
 
 services:
+  # Locally mocked aws services
+  aws:
+    image: localstack/localstack
+    ports:
+      - 4572:4572
+      - 8081:8080
```

Comment:
**Note:** While running in development, navigating to `localhost:8081` will bring up a web UI showing the aws services visually.

Not super useful, but it's there.

Reply:
Is 4572 the port S3 is being run on?

Reply:
Yes